var/spack/repos/builtin/packages/jali/package.py

```diff
@@ -18,7 +18,7 @@ class Jali(CMakePackage):
     maintainers = ['raovgarimella']
 
     version('master', branch='master')
-    version('1.1.6', sha256='1321c0ca847bb74f760398a25eeb640d09f87764d3527737d9b84d7ce601d163')
+    version('1.1.6', sha256='a2f4e4f238c60ea78486e0c9ea5b3e2cdd9d91c2ae5ea006a1d33a12e9eafa3a')
```

Comment:
Why did this sha change? It's very bad from a reproducibility and security standpoint. @raovgarimella did the contents change, the distribution method, ...?

Reply:
When trying to build this error message was given "expected 1321c0ca... but got a2f4e4f....."

Reply:
@sethrj It was my fault. I did a manual unzip and tarring and must have gotten the sha sum wrong. It went undetected until @Tracy-Pantleo tested it.